src/System.Reflection.Emit.Lightweight/ref/System.Reflection.Emit.Lightweight.cs

```diff
@@ -8,6 +8,29 @@
 
 namespace System.Reflection.Emit
 {
+    public sealed partial class DynamicILInfo
+    {
+        internal DynamicILInfo(object scope, DynamicMethod method, byte[] methodSignature) { }
+        public System.Reflection.Emit.DynamicMethod DynamicMethod { get { throw null; } }
+        public int GetTokenFor(byte[] signature) { throw null; }
+        public int GetTokenFor(DynamicMethod method) { throw null; }
+        public int GetTokenFor(RuntimeFieldHandle field) { throw null; }
+        public int GetTokenFor(RuntimeFieldHandle field, RuntimeTypeHandle contextType) { throw null; }
+        public int GetTokenFor(RuntimeMethodHandle method) { throw null; }
+        public int GetTokenFor(RuntimeMethodHandle method, RuntimeTypeHandle contextType) { throw null; }
+        public int GetTokenFor(RuntimeTypeHandle type) { throw null; }
+        public int GetTokenFor(string literal) { throw null; }
+        [System.CLSCompliantAttribute(false)]
```

Comment:
@jkotas what is the current thinking about use of CLSCompliant? I thought we didn't care about it any more, but perhaps there is some reason for keeping it?

Reply:
CoreFX public surface is annotated with the CLSCompliant. There are a few exceptions that have `<ClsCompliant>false</ClsCompliant>`.

IIRC, the last time this question was asked, FXDC said that we still care about this.

Reply:
Btw: the C# compiler doesn't care about CLS compliance that much anymore either: dotnet/Roslyn#26760

Reply:
@terrajobst?